internal/config/config.go

```diff
@@ -51,6 +51,9 @@ type Config struct {
 
 	EnableGraphql          bool `default:"TRUE" desc:"*** Enable GraphQL Endpoint" split_words:"true"`
 	GraphqlWebClientEnable bool `default:"TRUE" desc:"Enable GraphiQL, the GraphQL web client" split_words:"true"`
+
+	EnableWitnessDistro           bool   `default:"FALSE" desc:"*** Enable Witness Distribution Endpoints" split_words:"true"`
+	WitnessDistroManifestLocation string `default:"/tmp/witness/manifest.yaml" desc:"Location of the manifest describing available versions of witness"`
```

Comment:
`WitnessDistroManifestLocation` => `WitnessDistroConfig` ? It's at least a little bit shorter.